services/learn-card-network/brain-service/test/boosts.spec.ts

```diff
@@ -546,6 +623,340 @@ describe('Boosts', () => {
+            ).not.toMatchObject(beforeUpdateBoostCredential);
+        });
+
+        it('should not allow non-admins to update boost credential', async () => {
+            const uri = await userA.clients.fullAuth.boost.createBoost({
+                credential: testVc,
+                status: BoostStatus.enum.DRAFT,
+            });
+
+            const beforeUpdateBoostCredential = await userA.clients.fullAuth.storage.resolve({
+                uri,
+            });
+            await expect(
+                userB.clients.fullAuth.boost.updateBoost({
+                    uri,
+                    updates: { credential: testUnsignedBoost },
+                })
+            ).rejects.toMatchObject({ code: 'UNAUTHORIZED' });
+
+            const newBoosts = await userA.clients.fullAuth.boost.getBoosts();
+            const newBoost = newBoosts[0]!;
+
+            expect(
+                await userA.clients.fullAuth.storage.resolve({ uri: newBoost.uri })
+            ).toMatchObject(beforeUpdateBoostCredential);
+        });
+    });
+
+    describe('getBoostAdmins', () => {
+        beforeEach(async () => {
+            await Profile.delete({ detach: true, where: {} });
+            await Boost.delete({ detach: true, where: {} });
+            await userA.clients.fullAuth.profile.createProfile({ profileId: 'usera' });
+            await userB.clients.fullAuth.profile.createProfile({ profileId: 'userb' });
+        });
+
+        afterAll(async () => {
+            await Profile.delete({ detach: true, where: {} });
+            await Boost.delete({ detach: true, where: {} });
+        });
+
+        it('should require full auth to get boost admins', async () => {
+            const uri = await userA.clients.fullAuth.boost.createBoost({
+                credential: testUnsignedBoost,
+            });
+            await userA.clients.fullAuth.boost.getBoostAdmins({ uri });
+
+            await expect(noAuthClient.boost.getBoostAdmins({ uri })).rejects.toMatchObject({
+                code: 'UNAUTHORIZED',
+            });
+            await expect(
+                userA.clients.partialAuth.boost.getBoostAdmins({ uri })
+            ).rejects.toMatchObject({
+                code: 'UNAUTHORIZED',
+            });
+        });
+
+        it('should allow getting boost admins', async () => {
+            const uri = await userA.clients.fullAuth.boost.createBoost({
+                credential: testUnsignedBoost,
+            });
+            await expect(
+                userA.clients.fullAuth.boost.getBoostAdmins({ uri })
+            ).resolves.not.toThrow();
+
+            expect(
+                (await userA.clients.fullAuth.boost.getBoostAdmins({ uri })).records
+            ).toHaveLength(0);
+
+            await userA.clients.fullAuth.boost.addBoostAdmin({ uri, profileId: 'userb' });
+
+            expect(
+                (await userA.clients.fullAuth.boost.getBoostAdmins({ uri })).records
```

Comment:
Is this 1 because of 'include self' param? So the count is just for userb?

Reply:
Yup! I wasn't sure if `includeSelf` makes more sense as true or false as a default, so I decided to go with excluding self by default